src/proc_image_processing/cpu/algorithm/major_edge_extractor.h

```diff
@@ -124,7 +124,7 @@ namespace proc_image_processing {
 
     inline bool MajorEdgeExtractor::isValueConnected(RefPointPtr ref, float value) {
         if (ref != nullptr)
-            return float(ref->_pix_value) * PERCENT_OF_VAL_FOR_VALUE_CONNECTION <= value;
+            return ref->_pix_value * PERCENT_OF_VAL_FOR_VALUE_CONNECTION <= value;
```

Comment:
nice catch on the unneeded conversion. 
I guess the compiler just ignore those, since it's pointless ton convert to the same type as it is..